frontend/src/apis/buybacks/BuybacksConversions.tsx

```diff
@@ -45,6 +47,8 @@ export function APIToInternalBBConversion(bb: APIBB): BuyBack {
     vendorName: bb.vendor_name,
     sales: sales,
     isDeletable: bb.is_deletable,
+    associatedUserId: bb.user,
+    associatedUserName: bb.username,
```

Comment:
creatorId and creatorName work. I will change it to that instead. I think I am going to keep the Id because know the user and having both username and id is vital as future features can use these two variables in their own right. I am also currently sorting based on userId since sorting for it is implemented, but not for the username. The sortfield is ["associatedUserName", "user"] to correct for this lack of sorting.